--- Qt/qt-everywhere-opensource-src-4.8.6/src/gui/dialogs/qfiledialog.cpp.orig	2015-03-16 18:30:17.254265813 +0100
+++ Qt/qt-everywhere-opensource-src-4.8.6/src/gui/dialogs/qfiledialog.cpp	2015-03-16 18:30:24.878265495 +0100
@@ -58,7 +58,7 @@
 #include <qdebug.h>
 #include <qapplication.h>
 #include <qstylepainter.h>
-#include <itemviews/qfileiconprovider_p.h>
+#include "../itemviews/qfileiconprovider_p.h"
 #if !defined(Q_WS_WINCE) && !defined(Q_OS_SYMBIAN)
 #include "ui_qfiledialog.h"
 #else
--- Qt/qt-everywhere-opensource-src-4.8.6/src/3rdparty/webkit/Source/WebKit/qt/tests/hybridPixmap/hybridPixmap.pro.orig	2014-04-10 20:37:12.000000000 +0200
+++ Qt/qt-everywhere-opensource-src-4.8.6/src/3rdparty/webkit/Source/WebKit/qt/tests/hybridPixmap/hybridPixmap.pro	2015-03-27 15:56:17.801915700 +0100
@@ -9,3 +9,4 @@ HEADERS += widget.h
 FORMS += widget.ui
 RESOURCES += resources.qrc
 CONFIG += console
+INCLUDEPATH += .
